question-servers/freeform.js

```diff
@@ -106,6 +106,13 @@ module.exports = {
                             // TODO remove once everyone is using the new version
                             if (elementType === 'core') {
                                 elements[elementName.replace(/-/g, '_')] = info;
+                                
+                                if ('additionalNames' in info) {
+                                    info.additionalNames.forEach(name => {
+                                        elements[name] = info;
+                                        elements[name.replace(/-/g, '_')] = info;
```

Comment:
Is replacing `-` to `_` required here?

Reply:
Not sure, are underscores still in the process of being phased out?  I have it there just in case but I'm fine taking It out if we think it's unneeded.

Reply:
We have completely phased out underscores, but we will support them indefinitely for backwards compatibility so we want this code.